recipes/r-misha/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set version = "4.0.4" %}
+package:
+  name: 'r-misha'
+  version: '{{ version }}'
+
+source:
+  url: 'https://bitbucket.org/tanaylab/misha-package/get/96b4e57cfe08.zip'
+  sha256: f15608e7597085d62cbfc731b7936589e5d1f9938149104bd00cdc7d5f619c06
+
+build:
+  number: 0
+  skip: true # [osx)
```

Comment:
I think the typo in the closing bracket seems top have triggered a "silent" bug in the bioconda testing framework. I just had a quick glance at the circleci output and noticed that the last lines read:
```
10:01:21 BIOCONDA INFO Nothing to be done for recipe recipes/r-misha
10:01:21 BIOCONDA INFO BUILD SUMMARY: successfully built 0 of 1 recipes
```

So it seems like the test skipped all builds. As it's already merged, I would suggest fixing by correcting to the `]` bracket and updating the `build` `number` to `1`.

Reply:
thanks for catching this! Should be fixed now.

M